sklearn/utils/_repr_html/estimator.js

```diff
@@ -43,6 +50,46 @@ document.querySelectorAll('.copy-paste-icon').forEach(function(element) {
 });
 
 
+function copyRowsToClipboard(element) {
```

Comment:
I would rename this function `copyFeatureNamesToClipboard` to be more explicit. We can also start to document the function. Something like this maybe:


```suggestion
/**
 * Copy the list of feature names formatted as a Python list.
 *
 * @param {HTMLElement} element - The copy button inside a `.features` block; its siblings
 *   contain a `details` element and a table containing feature named.
 * @returns {boolean} Always returns `false` so callers can prevent the default click behavior.
 */
function copyFeatureNamesToClipboard(element) {
```

Reply:
I also think that we should rename the function above `copyToClipboard` to `copyParameterNameToClipboard` to be more explicit of what it does.

Reply:
In addition, we merged allowing to write unit test for the javascript, e.g. https://github.com/scikit-learn/scikit-learn/pull/32345/files#diff-5a1182f734949d095df0be4405392b4ba589ac71e6d64fb840034b9c65c79f76R81

I think that we could make a similar test for this function.

Reply:
Done.